scripts/busybox-updater.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# This script is called via .github/workflows/container-version.yaml
+# No need to manually run this (unless you want to force an update NOW)
+
+# Get the tags from the registry, so we can get the base manifest_digest ID
+echo "Doing first CURL request"
```

Comment:
Nit: maybe specify what each call is doing e.g.:
```suggestion
echo "Getting latest tags"
```